### PR TITLE
Adds two libraries, Google Benchmark  and F2C

### PR DESCRIPTION
**Pull request by mdavezac**

### CODE REVIEW

File: cmake/projects/F2C/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+include(hunter_configuration_types)
+
+# Makes it possible to use syste cfitsio
+hunter_add_version(
+    PACKAGE_NAME
+    F2C
+    VERSION
+    "0.0"
+    URL
+    "http://netlib.sandia.gov/f2c/libf2c.zip"
```
Comment (ruslo) [inline cmake/projects/F2C/hunter.cmake]: This look like an unstable link, probably it should be something like `http://netlib.sandia.gov/f2c/libf2c-v0.0.zip` ?

File: cmake/projects/F2C/schemes/F2C.cmake.in [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2016 University College London
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include(ExternalProject) # ExternalProject_Add
+
+# Scheme for CMake projects
```
Comment (ruslo) [inline cmake/projects/F2C/schemes/F2C.cmake.in]: This comment doesn't belong here.

File: cmake/projects/F2C/schemes/F2C.cmake.in [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        "@CMAKE_COMMAND@" -E make_directory "@HUNTER_PACKAGE_INSTALL_PREFIX@/lib"
+      COMMAND
+        "@CMAKE_COMMAND@" -E copy f2c.h "@HUNTER_PACKAGE_INSTALL_PREFIX@/include"
+      COMMAND
+        "@CMAKE_COMMAND@" -E copy libf2c.a "@HUNTER_PACKAGE_INSTALL_PREFIX@/lib"
+      LOG_BUILD ${log_build}
+      LOG_INSTALL ${log_install}
+  )
+
+  # Each external project must depends on previous one since they all use
```
Comment (ruslo) [inline cmake/projects/F2C/schemes/F2C.cmake.in]: This one not used and can be removed

File: cmake/projects/GBenchmark/hunter.cmake [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    VERSION
+    "1.1.0"
+    URL
+    "https://github.com/google/benchmark/archive/v1.1.0.tar.gz"
+    SHA1
+    8c539bbe2a212618fa87b6c38fba087100b6e4ae
+)
+
+hunter_cmake_args(GBenchmark CMAKE_ARGS BENCHMARK_ENABLE_TESTING=OFF)
+hunter_configuration_types(GBenchmark CONFIGURATION_TYPES Release)
```
Comment (ruslo) [inline cmake/projects/GBenchmark/hunter.cmake]: Why `Release` only?

Comment (mdavezac) [inline cmake/projects/GBenchmark/hunter.cmake]: Because it is a benchmark library so compiling it in Debug doesn't really make any sense.

**Comment by mdavezac** [comment]
I'll resubmit this as two pull-requests

**Pull request closed without merging**